frontend/tests/features/wheresMyGene/landingPageFilter.test.ts

```diff
@@ -0,0 +1,117 @@
+/**
+ * Test suite for select filter-related utils.
+ */
+import { expect, test } from "@playwright/test";
+import { TEST_URL } from "tests/common/constants";
+
+const { describe } = test;
+const CHEVRON_LEFT = '[data-icon="chevron-left"]';
+const FIRST_OPTION = '[data-option-index="0"]';
+const DATASET_FILTER = "dataset-filter";
+
+describe("Left side bar", () => {
+  test.beforeEach(async ({ page }) => {
```

Comment:
let's add

```ts
skip(!isDevStagingProd, "WMG BE API does not work locally or in rdev");
```

Since these tests actually only work when connected to dev, staging, and prod BE APIs

Reply:
Thanks for extracting the function `goToWMG` 🙏 

Can we use `goToWMG` directly in each test block and not use `beforeEach` at all?

E.g.,

```ts
function goToWMG({ page }) {
  return Promise.all([
      page.waitForResponse(
        (resp: { url: () => string | string[]; status: () => number }) =>
          resp.url().includes("/wmg/v1/filters") && resp.status() === 200
      ),
      page.goto(`${TEST_URL}${ROUTES.WHERE_IS_MY_GENE}`),
    ]);
}

...

test("Should verify main panel components", async ({ page }) => {
  await goToWMG({ page });
  
  // +Tissue button
  await expect(page.getByTestId(ADD_TISSUE).first()).toBeVisible();

  ...
})
```

Following this best practice:
https://kentcdodds.com/blog/avoid-nesting-when-youre-testing

Thank you!

Reply:
Hey @victorikeokoro6 I think we're still using beforeEach here and the goal is to not use beforeEach. PTAL thank you :pray:

Reply:
sorry unresolving this since we're still using `beforeEach` and it's not following the best practice shared here:

https://kentcdodds.com/blog/avoid-nesting-when-youre-testing 🙏

Reply:
@tihuan  I have resolved this now

Reply:
I might be missing something, but `test.beforeEach(async ({ page }) => {` is still in this PR 😆

Reply:
@charles-testco I still see `beforeEach()` in this PR, how is this resolved? Thank you!